### PR TITLE
Small example of making a request using curl

### PR DESCRIPTION
**Pull request by hamiltont**
Not sure if this is useful enough to be included, but it would have saved me about 2min since i thought the API was POST-only

### CODE REVIEW

**Comment by hamiltont** [comment]
ps - targeted dev branch. Is that correct?

File: API.md [hunk 1]
```diff
@@ -30,6 +30,24 @@ General optional parameters:
     debug:      1
 ```
 
+### Request Example
+
+Making a request to PlayPy using the [curl](https://curl.haxx.se/) tool is straightforward
```
Comment (Arcanemagus) [inline API.md:35]: * `PlayPy` -> `PlexPy`
* Put a `:` at the end of the line

File: API.md [hunk 2]
```diff
@@ -30,6 +30,24 @@ General optional parameters:
...
+
+```
```
Comment (Arcanemagus) [inline API.md:37]: If you set the language to `ShellSession` it will do some highlighting for you:

\`\`\`ShellSession

File: API.md [hunk 3]
```diff
@@ -30,6 +30,24 @@ General optional parameters:
...
+your_name$ curl "http://<hostname>:<port>/api/v2?apikey=<apikey>&cmd=get_home_stats"  \
```
Comment (Arcanemagus) [inline API.md:38]: The `your_name` is implied from the `$`, no need to include it.

**Review by Arcanemagus — CHANGES_REQUESTED** [review]

**Comment by Hellowlol** [comment]
There shouldn't be any direct pulls to the API md. The page is made by the API. See https://github.com/JonnyWong16/plexpy/blob/master/plexpy/api2.py#L344

Adding debug=1 will pretty print and sort the keys for you.
When I think about it, it would be cool to have a API builder like sickbeard did.

**Comment by hamiltont** [comment]
oop, good call @Hellowlol . I'll just close this PR and may open a new one later on that file

**Pull request closed without merging**